### PR TITLE
WC add explainer on connection timeout

### PR DESCRIPTION
**Pull request by estebanmino**
Add an explainer sheet when users wait up to 10 sec to get a connection request

https://recordit.co/7eKvucH42b

### CODE REVIEW

File: src/screens/ExplainSheet.js [hunk 1]
```diff
@@ -116,6 +116,13 @@ export const explainers = {
     readMoreLink: 'https://polygon.technology/',
     extraHeight: 120,
   },
+  wallet_connect: {
```
Comment (skylarbarrera) [inline src/screens/ExplainSheet.js:119]: can we change the name of this to failed_wc_connection or something more semantic?

Comment (estebanmino) [reply src/screens/ExplainSheet.js]: changed

**Review by skylarbarrera — APPROVED** [review]
🥑

File: src/redux/walletconnect.js [hunk 2]
```diff
@@ -149,13 +150,19 @@ export const walletConnectOnSessionRequest = (uri, callback) => async (
...
+            callback?.('reject', dappScheme);
             analytics.track('Rejected new WalletConnect session', {
               dappName,
               dappUrl,
             });
+          } else {
+            callback?.('timedOut', dappScheme);
+            analytics.track('New WalletConnect session time out', {
+              dappName,
+              dappUrl,
```
Comment (brunobar79) [inline src/redux/walletconnect.js:164]: Can we add the bridge url here? I think that would be super useful

**Review by brunobar79 — APPROVED** [review]
💯

File: src/redux/walletconnect.js (hunk 2 above)
Comment (estebanmino) [reply src/redux/walletconnect.js:168]: 💯

**Pull request merged**